packages/@dcl/sdk-commands/src/logic/bundle.ts

```diff
@@ -203,6 +204,12 @@ function compositeLoader(components: BundleComponents, options: CompileOptions):
             )
             contents = `export const compositeFromLoader = {${data.compositeLines.join(',')}}`
             watchFiles = data.watchFiles
+
+            if (data.withErrors) {
+              components.logger.log(
```

Comment:
we have a visually impacting "beautiful logger" for errors if you'd like to use it

Reply:
I'll do the improvement later